rust/src/clib/lib.rs

```diff
@@ -303,6 +305,40 @@ pub extern "C" fn nmstate_checkpoint_rollback(
     }
 }
 
+#[allow(clippy::not_unsafe_ptr_arg_deref)]
+#[no_mangle]
+pub extern "C" fn nmstate_nm_version(
```

Comment:
Please use `nmstate_backend_versions()` as nmstate is design to hide the backend.